homeassistant/__main__.py

```diff
@@ -256,42 +250,61 @@ def open_browser(event):
 
         hass.bus.listen_once(EVENT_HOMEASSISTANT_START, open_browser)
 
+    print('Starting Home-Assistant')
```

Comment:
This is after we have loaded the configuration and are about to start the loop where we set up handlers and respond to SIGINT/SIGHUP/SIGTERM signals. It was helpful for me to know but I can remove it.